src/Symfony/Component/Form/Tests/Extension/Core/Type/DateTypeTest.php

```diff
@@ -353,7 +373,7 @@ public function testThrowExceptionIfFormatIsNoPattern()
 
     /**
      * @expectedException \Symfony\Component\OptionsResolver\Exception\InvalidOptionsException
-     * @expectedExceptionMessage The "format" option should contain the letters "y", "M" and "d". Its current value is "yy".
+     * @expectedExceptionMessage The DateTime format "yy" is not supported in ICU. Please be aware that the data may not reflect the format.
```

Comment:
Does this really need to change (same below?) Is it really legitimate?

Reply:
I think it is because if you pass 'yy', the message is wrong, IMO.

Reply:
This is right, yy is not valid in ICU.